pkg/validation/internal/operatorhub.go

```diff
@@ -20,48 +20,58 @@ import (
 var OperatorHubValidator interfaces.Validator = interfaces.ValidatorFunc(validateOperatorHub)
 
 var validCapabilities = map[string]struct{}{
-	"Basic Install":     struct{}{},
-	"Seamless Upgrades": struct{}{},
-	"Full Lifecycle":    struct{}{},
-	"Deep Insights":     struct{}{},
-	"Auto Pilot":        struct{}{},
+	"Basic Install":     {},
+	"Seamless Upgrades": {},
+	"Full Lifecycle":    {},
+	"Deep Insights":     {},
+	"Auto Pilot":        {},
 }
 
 var validMediatypes = map[string]struct{}{
-	"image/gif":     struct{}{},
-	"image/jpeg":    struct{}{},
-	"image/png":     struct{}{},
-	"image/svg+xml": struct{}{},
+	"image/gif":     {},
+	"image/jpeg":    {},
+	"image/png":     {},
+	"image/svg+xml": {},
 }
 
 var validCategories = map[string]struct{}{
-	"AI/Machine Learning":    struct{}{},
-	"Application Runtime":    struct{}{},
-	"Big Data":               struct{}{},
-	"Cloud Provider":         struct{}{},
-	"Developer Tools":        struct{}{},
-	"Database":               struct{}{},
-	"Integration & Delivery": struct{}{},
-	"Logging & Tracing":      struct{}{},
-	"Monitoring":             struct{}{},
-	"Networking":             struct{}{},
-	"OpenShift Optional":     struct{}{},
-	"Security":               struct{}{},
-	"Storage":                struct{}{},
-	"Streaming & Messaging":  struct{}{},
+	"AI/Machine Learning":    {},
+	"Application Runtime":    {},
+	"Big Data":               {},
+	"Cloud Provider":         {},
+	"Developer Tools":        {},
+	"Database":               {},
+	"Integration & Delivery": {},
+	"Logging & Tracing":      {},
+	"Monitoring":             {},
+	"Networking":             {},
+	"OpenShift Optional":     {},
+	"Security":               {},
+	"Storage":                {},
+	"Streaming & Messaging":  {},
 }
 
 func validateOperatorHub(objs ...interface{}) (results []errors.ManifestResult) {
+
+	k8sVersion := ""
+	for _, obj := range objs {
```

Comment:
Why do this here and overload the validate method? I'm wondering if this should just happen further into the validation method that cares about it.

Reply:
It is inside of the only validator that uses it. See that it is inside of `validateOperatorHub` and will not be done for any other validator. Note that in SDK we will add a flag `--optional-values="k8s=1.22"` to pass optional values. So, if a specific validator requires/uses any optional value we would be able to get that. 

We might want other keys=values in the future.

Reply:
See: (These changes tested/checked with SDK)

<img width="1098" alt="Screen Shot 2021-03-24 at 00 20 55" src="https://user-images.githubusercontent.com/7708031/112235598-e26f4e00-8c36-11eb-991d-38d710377309.png">

See the help: 

<img width="1099" alt="Screen Shot 2021-03-24 at 00 22 26" src="https://user-images.githubusercontent.com/7708031/112235664-0763c100-8c37-11eb-9aea-b01e3c70e571.png">

Reply:
It is inside of the `ValidateOperatorHub` only. The idea here is to allow us to be able to pass a map[string] since for the CLI we need a generic flag option which indeed can be used by any other optional validator. We might want to inform other optional values in the future as well.   

I added how it will be used in SDK. See: https://github.com/operator-framework/api/pull/103#issue-599058459. I hope that clarifies the motivations. 

Please, let me know if it answers your question.

Reply:
Closing that as solved. However, feel free to re-open if you see that is required.